ui/src/components/Home/VerificationSection/CameraAccessDenied.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import {ReactComponent as NoPhotographyIcon} from "../../../assets/no-photography-icon.svg";
+import { NoPhotographyIcon } from '../../../utils/theme-utils';
```

Comment:
Can we rename this to `CameraAccessDeniedIcon` as that is what our use case also?